src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.State.cs

```diff
@@ -485,6 +485,27 @@ internal bool IsBound
             aliasesOfReferencedAssemblies = aliasesOfReferencedAssembliesBuilder.ToImmutableAndFree();
         }
 
+        internal ImmutableDictionary<AssemblyIdentity, AssemblyIdentity> GetAssemblyReferenceIdentityMap(ImmutableArray<AssemblyIdentity> originalIdentities, ImmutableArray<TAssemblySymbol> symbols)
```

Comment:
Please add a comment describing what map does this method build. What values are included and what values aren't included?